test/common/router/config_impl_test.cc

```diff
@@ -3166,6 +3166,12 @@ TEST_F(RouterMatcherHashPolicyTest, HashIpv4DifferentAddresses) {
 }
 
 TEST_F(RouterMatcherHashPolicyTest, HashIpv6DifferentAddresses) {
+  if (!TestEnvironment::shouldRunTestForIpVersion(Network::Address::IpVersion::v6)) {
+    // an exception like "IPv6 addresses are not supported on this machine" will be thrown
+    // if we don't add IP version check and run test on a machine that doesn't support IPv6
+    return;
```

Comment:
we should use `GTEST_SKIP` here, something like `GTEST_SKIP() << the machine aren't support IPv6`

Reply:
@soulxu 
Thanks. Update according to the review. The CI failure seems to be caused by failing to upload the docker image: https://dev.azure.com/cncf/4684fb3d-0389-4e0b-8251-221942316e06/_apis/build/builds/166826/logs/492